docs/examples/compose/compose.yaml

```diff
@@ -0,0 +1,78 @@
+services:
+  postgresql:
+    image: postgres:16
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -d $${POSTGRES_DB} -U $${POSTGRES_USER}"]
+      interval: 1s
+      timeout: 2s
+      retries: 300
+    env_file:
+    - env.d/postgresql
+    - env.d/common
+    environment:
+    - PGDATA=/var/lib/postgresql/data/pgdata
+    volumes:
+    - ./data/databases/backend:/var/lib/postgresql/data/pgdata
+
+  redis:
+    image: redis:5
```

Comment:
Am I mistaken or is Redis 5 EOL since a long time?

https://redis.io/docs/latest/operate/rs/installing-upgrading/product-lifecycle/

Given the new license of Redis it might also make sense to use Valkey by default instead?